src/main/kotlin/fi/metatavu/lipsanen/projects/milestones/tasks/TaskEntity.kt

```diff
@@ -32,6 +33,22 @@ class TaskEntity : Metadata() {
     @Enumerated(EnumType.STRING)
     lateinit var status: TaskStatus
 
+    @Column(columnDefinition = "json")
+    lateinit var assigneeIds: List<String>
+
+    @Enumerated(EnumType.STRING)
+    @Column(nullable = true)
+    var userRole: UserRole? = null
+
+    @Column(nullable = true)
+    var estimatedDuration: String? = null
+
+    @Column(nullable = true)
+    var estimatedReadiness: String? = null
+
+    @Column(columnDefinition = "json")
+    lateinit var attachmentUrls: List<String>
```

Comment:
Questionable

Reply:
Could you add it as separate entity instead?